src/schema/v2/partner.ts

```diff
@@ -157,6 +158,72 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      allArtistsConnection: {
```

Comment:
*Sort* of odd that this is even a connection if it *always* returns all the results, non-paginated.

Reply:
Though I guess it makes sense if you want to access the `totalCount`, for instance